R/observer.R

```diff
@@ -1,35 +0,0 @@
-#' Register an external MLflow observer
-#'
-#' Registers an external MLflow observer that will receive a
-#' `register_tracking_event(event_name, data)` callback on any model tracking
-#' event such as "create_run", "delete_run", or "log_metric".
-#' Each observer should have a `register_tracking_event(event_name, data)`
-#' callback accepting a character vector `event_name` specifying the name of
-#' the tracking event, and `data` containing a list of attributes of the event.
-#' The callback should be non-blocking, and ideally should complete
-#'  instantaneously. Any exception thrown from the callback will be ignored.
-#'
-#' @param observer The observer object (see example)
-#' @export
-register_external_observer <- function(observer) {
```

Comment:
i'm fine with getting rid of this. this seems very niche and is sort of against the opinionated nature of the package

Reply:
Yeah I have no idea what this even does